addons/project/static/tests/view/project_task_kanban.test.js

```diff
@@ -0,0 +1,41 @@
+import { expect, test, describe } from "@odoo/hoot";
+import { animationFrame } from "@odoo/hoot-dom";
+
+import { onRpc, mountView } from "@web/../tests/web_test_helpers";
+
+import { defineProjectModels } from "../project_models";
+
+defineProjectModels();
+describe.current.tags("desktop");
+
+const viewParams = {
+    resModel: "project.task",
+    type: "kanban",
+    arch: `
+        <kanban js_class="project_task_kanban">
+            <templates>
+                <t t-name="card">
+                    <field name="name"/>
+                </t>
+            </templates>
+        </kanban>`,
+    context: {
+        active_model: "project.project",
+        default_project_id: 1,
+    },
+    groupBy: ["stage_id"],
+};
+
+test("quick create button is visible when the user has access rights.", async () => {
```

Comment:
@maad-odoo  is there any specific reason to create a new file
you can move these tests to [this file](https://github.com/odoo/odoo/blob/18.0/addons/project/static/tests/project_task_kanban_view.test.js) to reduce the reusability of code.